### DISCUSSION TITLE
New Project Idea and Questions about Community Members

### DISCUSSION THREAD

**Discussion by Tbr00ksy** [Green-Software-Foundation/discussions#109] [Ask Anything]
Hi @jawache and @seanmcilroy29 and the rest of the GSF Community,

My name is Tim Brooks and I currently hold the position of Senior Product Manager for IBM Z AIOps Software. I am reaching out to express my genuine interest in the work that GSF is undertaking. Promoting sustainability within the software industry is truly commendable, and I am eager to contribute to the cause. 

Right now I am reaching out as an individual contributor trying to learn if IBM has any history in this community, and if so, if you are aware of any contacts I could reach out to learn more about their current involvement. 

Overall, I am working with a group of IBMers to try and measure the carbon footprint for software Development. I thought that engaging and collaborating on this effort in the open source community could be more productive that trying to answer the question "how sustainable are my software development practices" in isolation.  

I look forward to hearing back from you and hope to be able to connect. 

You can find me on LI here: https://www.linkedin.com/in/tim-dk-brooks/

Best,
Tim

**Comment by Tbr00ksy** [comment]
@russelltrow any advice for getting started and how we can partner in this community?

**Comment by seanmcilroy29** [comment]
@Tbr00ksy - Thanks Tim for your submission.
Just to let you know, I'm in ongoing discussions with representatives from IBM regarding your organization's potential membership in the GSF. Although the talks are in the initial stages, we are hopeful to have IBM as a member early in the upcoming year.

Unfortunately at the moment, the GSF membership framework doesn't support individual contributors.

**Reply by Tbr00ksy** [reply]
Hi @seanmcilroy29 ,

Thank you for your response. 

Since you are still in the initial stages regarding IBM's membership, would you be willing to provide me with a name of the IBMer I can follow-up with internally on my side? The IBM team that I'm currently working with is not aware of anyone in IBM that in working with the GSF today, and by showing a broader interest across IBM we can help drive the case for why we should join as a member org.

If you don't feel comfortable posting your IBM contact here, you could send me a DM in LI?

Thanks,
Tim

**Reply by jawache** [reply]
Hi @Tbr00ksy feel free to email sean@greensoftware.foundation so we can discuss in private :)
